src/utils/artifact/generateArtifacts.ts

```diff
@@ -20,6 +24,7 @@ export async function generateArtifacts(
   bre: BuidlerRuntimeEnvironment
 ): Promise<void> {
   const arapp = readArapp()
+  const appName = parseAppName(arapp, bre.network.name)
```

Comment:
It looks like the function signature changed here between the previous usage (it didn't need the `bre.network.name`). Is this to select the right app name based on the environment?

Reply:
Yes, we were not handling the right `appName` during publication for each environment